aws/signer/v4/handler_test.go

```diff
@@ -0,0 +1,172 @@
+package v4
```

Comment:
nit probably want to rename this file to middleware_test or signing_middleware_test